src/jdk.incubator.foreign/share/classes/jdk/incubator/foreign/CLinker.java

```diff
@@ -196,7 +196,8 @@ static CLinker getInstance() {
      * @param type      the method type.
      * @param function  the function descriptor.
      * @return the downcall method handle.
-     * @throws IllegalArgumentException in the case of a method type and function descriptor mismatch.
+     * @throws IllegalArgumentException in the case of a method type and function descriptor mismatch, or if the symbol
+     *                                  is {@link MemoryAddress#NULL}
```

Comment:
Is the indentation deliberate? What about NPE?

Reply:
Actually, I think the NPE is covered by this comment on the class 

```
 * <p> Unless otherwise specified, passing a {@code null} argument, or an array argument containing one or more {@code null}
 * elements to a method in this class causes a {@link NullPointerException NullPointerException} to be thrown. </p>
 ```

Reply:
ah true